robotiq_c_model_control/src/robotiq_c_model_control/constants.py

```diff
@@ -0,0 +1,13 @@
+"""
+Copyright 2018 by READY Robotics Corporation.
+All rights reserved. No person may copy, distribute, publicly display, create derivative works from or otherwise
+use or modify this software without first obtaining a license from the READY Robotics Corporation.
+"""
+ORIGINAL = 'original'
+MODBUS = 'modbus'
+
+# The slave/device IDs where we expect to find grippers on the MODBUS
+VALID_DEVICE_IDS = (9, 15)
```

Comment:
I thought we were going to pick these up through deployment_configuration.yaml params?

Reply:
indeed. debating whether this is OOS for this release or this PR in general.

Reply:
Resolved as part of this PR.